src/matrix/matrixt.jl

```diff
@@ -0,0 +1,191 @@
+"""
+    MatrixT(ν, M, Σ, Ω)
```

Comment:
Should be `MatrixTDist` for consistency with `TDist`, `MvTDist`?

Reply:
I dig it.